mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/BookmarkFolderTitleView.java

```diff
@@ -0,0 +1,36 @@
+
+
+package org.mozilla.gecko.home;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.mozilla.gecko.R;
+
+/**
+ * Copyright © Cliqz 2018
+ */
+public class BookmarkFolderTitleView extends LinearLayout {
```

Comment:
why do you inherit from a LinearLayout, I don't see any special UI theme added here.
You can use a Linear layout object instead. could you explain more?

Reply:
Although it does not do much I thought it would be a good idea to abstract this to a class instead of using just a linearlayout because it is one of the view type in the bookmarks list.